stdlib/2and3/zipfile.pyi

```diff
@@ -22,47 +27,47 @@ class ZipFile:
     debug = ...  # type: int
     comment = ...  # type: bytes
     filelist = ...  # type: List[ZipInfo]
-    def __init__(self, file: Union[Text, IO[bytes]], mode: Text = ..., compression: int = ...,
+    def __init__(self, file: Union[_Path, IO[bytes]], mode: Text = ..., compression: int = ...,
                  allowZip64: bool = ...) -> None: ...
     def __enter__(self) -> ZipFile: ...
     def __exit__(self, exc_type: Optional[Type[BaseException]],
                  exc_val: Optional[BaseException],
                  exc_tb: Optional[TracebackType]) -> bool: ...
     def close(self) -> None: ...
-    def getinfo(self, name: Text) -> ZipInfo: ...
+    def getinfo(self, name: _Path) -> ZipInfo: ...
```

Comment:
Does this really take a Path? From the code it looks like it can only return str.

Reply:
It looks like you're right!

```
$ python3.6
Python 3.6.2+ (heads/master-dirty:404de642c0, Sep 15 2017, 15:52:01)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.37)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import zipfile
>>> import io
>>> z = zipfile.ZipFile(file=io.BytesIO(), mode="a")
>>> z.writestr("file.txt", "hello world")
>>> z.getinfo("file.txt")
<ZipInfo filename='file.txt' filemode='?rw-------' file_size=11>
>>> import pathlib
>>> z.getinfo(pathlib.Path("file.txt"))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/homebrew/Cellar/python36/3.6.2+_254.20170915/Frameworks/Python.framework/Versions/3.6/lib/python3.6/zipfile.py", line 1281, in getinfo
    'There is no item named %r in the archive' % name)
KeyError: "There is no item named PosixPath('file.txt') in the archive"
```

Where is a good place for tests?

Reply:
We don't currently support tests for typeshed stubs, unfortunately.